Aiolos/Aiolos/Sources/PanelAnimator.swift

```diff
@@ -75,12 +75,27 @@ final class PanelAnimator {
         guard let animationDelegate = self.panel.animationDelegate else { return }
         guard self.panel.isVisible else { return }
 
-        let transitionCoordinator = PanelTransitionCoordinator(animator: self)
+        let transitionCoordinator = PanelTransitionCoordinator(animator: self, direction: .vertical)
         animationDelegate.panel(self.panel, willTransitionFrom: oldMode, to: newMode, with: transitionCoordinator)
         if let contentViewController = self.panel.contentViewController as? PanelAnimationDelegate {
             contentViewController.panel(self.panel, willTransitionFrom: oldMode, to: newMode, with: transitionCoordinator)
         }
     }
+    
+    func notifyDelegateAboutMove(to frame: CGRect) -> Bool {
```

Comment:
did my patch rename this to `notify`? Sorry about that, I wanted to keep the `ask` prefix

```suggestion
    func askDelegateAboutMove(to frame: CGRect) -> Bool {
```